testssl.sh

```diff
@@ -10927,6 +10927,9 @@ run_fs() {
                          done
                          [[ $i -eq $high ]] && break
                          supported_curve[i]=true
+                         if [[  ! "$TLS_EXTENSIONS" == *supported_groups* ]]; then
```

Comment:
Note that since #2689, `$TLS_EXTENSIONS` is now an array, so this check will not work (it will only look at the first element in the array). So, this line should be changed to use `${TLS_EXTENSIONS[*]}`. For consistency with other checks for whether a string contains a certain substring, I would suggest changing this to:
```
[[ ! "${TLS_EXTENSIONS[*]}" =~ supported_groups ]]
```